docs/reference/remap/errors/109_cannot_abort_function.cue

```diff
@@ -4,10 +4,9 @@ remap: errors: "109": {
 	title:       "Cannot abort function"
 	description: """
 		A [function call expression](\(urls.vrl_expressions)#\(remap.literals.regular_expression.anchor)) can't end with
-		`!` unless it's _fallible_. If a function can't produce a runtime error, it doesn't have an abort variant
-		that ends with `!`.
+		`!` unless the function is fallible, that is, if it can produce a runtime error. If the function is infallible,
+		it doesn't have an abort variant that ends with `!`.
 		"""
-	rationale:   null
```

Comment:
From looking at the source, it looks like `109` is generated by the `parse_grok` function (the "cannot abort function" error is already documented elsewhere in this PR).

I would probably advocate we use a different starting number to document function-specific error codes (e.g. `8xx`), so maybe we should just remove 109 from the docs for now, until we've made that change?

Reply:
Removed